app/javascript/shared/mixins/messageFormatterMixin.js

```diff
@@ -5,7 +5,12 @@ export default {
   methods: {
     formatMessage(message, isATweet) {
       const messageFormatter = new MessageFormatter(message, isATweet);
-      return messageFormatter.formattedMessage;
+      DOMPurify.addHook('afterSanitizeAttributes', function(node) {
+        if ('target' in node) {
+          node.setAttribute('target', '_blank');
+        }
+      });
+      return DOMPurify.sanitize(messageFormatter.formattedMessage);
```

Comment:
@fayazara 

Can we do this inside `formattedMessage` getter in MessageFormatter?

Reply:
Resolved in https://github.com/chatwoot/chatwoot/pull/4110/commits/77f0df3af58f42093a617d37977e42c2cd48cc64